sqlx-macros/src/database/mssql.rs

```diff
@@ -3,6 +3,7 @@ use sqlx_core as sqlx;
 impl_database_ext! {
     sqlx::mssql::Mssql {
         bool,
+        u8,
```

Comment:
```suggestion
```

Listing this over `i8` is also a breaking change as it will cause any existing invocation of the query macros to switch from `i8` to `u8`. Since they're both considered compatible with the same types, listing it _after_ `i8` is pointless. We can fix the macros in the next major revision.

Reply:
Fixed!